src/pages/index.tsx

```diff
@@ -62,6 +61,16 @@ const AdminPage: NextComponentType = () => {
     Promise.all(promises).then(() => setLoading(false));
   });
 
+  const handleChange = () => async (): Promise<void> => {
+    await fetch('/api/config', {
+      method: 'POST',
+      body: JSON.stringify({}),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+  };
```

Comment:
This endpoint requires that `configKey` and `configValue` are in the body. So you'll need to modify this `handleChange` function to take a parameter and pass those in.
```suggestion
  const handleChange = (configItem: Config) => async (): Promise<void> => {
    const newValue = configItem.value === 'true' ? 'false' : 'true';

    await fetch('/api/config', {
      method: 'POST',
      body: JSON.stringify({
        configKey: configItem.key,
        configValue: newValue
      }),
      headers: {
        'Content-Type': 'application/json',
      },
    });
  };
```